src/main/java/maksym/fedorenko/bookstore/dto/BookSearchParametersDto.java

```diff
@@ -0,0 +1,35 @@
+package maksym.fedorenko.bookstore.dto;
+
+import com.querydsl.core.types.Predicate;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import java.math.BigDecimal;
+import lombok.Data;
+import maksym.fedorenko.bookstore.model.QBook;
+
+@Data
+public class BookSearchParametersDto {
```

Comment:
you may use records for DTO, since we're on Java 17 :)

up to you though

Reply:
No chance :)

Reply:
@RedArtelerist  now there is a chance :) 

update you `maven-checkstyle-plugin` to 3.3.0